src/Main.elm

```diff
@@ -38,33 +43,45 @@ type alias AccountId =
 
 
 type alias AppState =
-    { serverUrl : ServerUrl
-    , accountsDict : Dict AccountId Account
+    { accountsDict : Dict AccountId Account
     , accountsTree : Tree.Multitree Account
     }
 
 
-type NextPage
-    = NextIndexPage
-    | NextAccountPage AccountId
+type alias Config =
+    { navKey : Nav.Key
+    , serverUrl : ServerUrl
+    }
 
 
-type alias Model =
+type alias State =
```

Comment:
State vs AppState is very confusing. Maybe `State` -> `AppState` and `AppState` -> `ServerState`.

Reply:
Or `AppState` -> `AccountState` or something.

Reply:
Or `LoadableState`. Up to you.